### PR TITLE
update minitest syntax

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: test/docs/basics_test.rb [hunk 1]
```diff
@@ -98,7 +94,7 @@ class Execute < Trailblazer::Activity::Railway
       #:pay-explicit end
 
       #:render
-      puts Trailblazer::Developer.render(Execute)
+      Trailblazer::Developer.render(Execute)
```
Comment (yogeshjain999) [inline test/docs/basics_test.rb:97]: This call can be removed too.

File: test/docs/variable_mapping_test.rb [hunk 2]
```diff
@@ -35,21 +35,12 @@ def create_model(ctx, current_user:, **)
         #~io-steps end
       end
       #:io-ary-hash end
-
-      signal, (ctx, flow_options) = Activity::TaskWrap.invoke(A::Memo::Create, [{params: {id: 1}}, {}])
-      signal.inspect.must_equal %{#<Trailblazer::Activity::End semantic=:success>}
-      ctx.inspect.must_equal %{{:params=>{:id=>1}, :current_user=>\"User 1\", :model=>\"User 1\"}}
-=begin
```
Comment (yogeshjain999) [inline test/docs/variable_mapping_test.rb:42]: This comment block is needed actually, being used while compiling `trb-website` docs

File: test/docs/variable_mapping_test.rb [hunk 3]
```diff
@@ -35,21 +35,12 @@ def create_model(ctx, current_user:, **)
...
-ctx = {params: {id: 1}}
-
-signal, (ctx, flow_options) = Activity::TaskWrap.invoke(A::Memo::Create, [ctx, {}])
-
-ctx #=> {:params=>{:id=>1}, :current_user=>#<User ..>, :model=>#<Memo ..>}}
-#:io-call end
-=end
     end
 
+    signal, (ctx, flow_options) = Activity::TaskWrap.invoke(A::Memo::Create, [{params: {id: 1}}, {}])
```
Comment (yogeshjain999) [inline test/docs/variable_mapping_test.rb:40]: Or instead, how about we wrap this in between `#:io-call`  & `#:io-call end`

**Review by yogeshjain999 — APPROVED** [review]
Looks good overall, left some comments. Nice work @seuros 💯

File: test/docs/variable_mapping_test.rb (hunk 3 above)
Comment (seuros) [reply test/docs/variable_mapping_test.rb:40]: I wont have access to the _

File: test/docs/variable_mapping_test.rb (hunk 2 above)
Comment (seuros) [reply test/docs/variable_mapping_test.rb:42]: I see ! Thanks

**Pull request merged**